internal/gitea/gitea.go

```diff
@@ -0,0 +1,14 @@
+package gitea
+
+type RepoConfig struct {
+	MonorepoSources []string `mapstructure:"monorepo-folders"`
```

Comment:
Naming nit: I'd probably name this `MonorepoFolders` to match the YAML name. Any reason for it to be different? I'd suggest the same for `IncludeRepos` -> `Includes` and `IgnoredRepos` -> `Ignores` below, but don't feel strongly about that.

Reply:
No, that was a result of me vacillating over whether to use sources or folders and ending up with a mix.

The ignores matches the GitHub code, but since we're moving away from copying that style exactly, I agree it would be better just as ignores. (Includes was aligned with that).